pilot/bin/push-docker

```diff
@@ -18,11 +18,12 @@
 #
 # This script creates a base image for the init image that includes iptables.
 #
-
 set -ex
-set -o errexit
-set -o nounset
-set -o pipefail
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Comment:
what is this doing? Some of our scripts have a simpler SCRIPTDIR thing that you could use.